.github/workflows/ci-cargo-audit.yml

```diff
@@ -32,4 +32,4 @@ jobs:
       # Run cargo audit using args from .cargo/audit.toml (ignores, etc.)
       - name: Run Cargo Audit
         # Ignoring warnings from solana_program_test dependencies which are only in dev dependencies
-        run: cargo audit -c always --ignore RUSTSEC-2021-0139 --ignore RUSTSEC-2023-0001 --ignore RUSTSEC-2020-0071 --ignore RUSTSEC-2023-0034
+        run: cargo audit -c always --ignore RUSTSEC-2023-0001 --ignore RUSTSEC-2020-0071
```

Comment:
Turns out updating the cargo.lock dependencies resolves a bunch of RUSTSEC errors